### PR TITLE
Ruby Nested Collections: Fix Additional Resources text

### PR DESCRIPTION
**Pull request by SpookyKasper**
Additional Resources section is mentioning existence of helpful links to other content whereas there is none at the moment. 
I updated the text to make it clear and ask for suggestions :)

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

File: ruby/basic_ruby/nested_collections.md [hunk 1]
```diff
@@ -328,7 +328,9 @@ Amazing! We have found a great solution to returning an array that only contains
 </div>
 
 ### Additional Resources
-This section contains helpful links to other content. It isn't required, so consider it supplemental.
+This section doesn't contains helpful links to other content yet! 
+Any suggestions ?
+Contribute on Github!
```
Comment (01zulfi) [inline ruby/basic_ruby/nested_collections.md:333]: We use specific wording for cases like this. It is outlined in our [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md).

The Additional Resources section should look like:

```suggestion

This section contains helpful links to related content. It isn’t required, so consider it supplemental.

*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
```

**Review by 01zulfi — CHANGES_REQUESTED** [review]
Thanks for wanting to contribute! I've left a requested change below.

**Comment by wise-king-sullyman** [comment]
@SpookyKasper are you planning to make the requested change?

**Comment by ChargrilledChook** [comment]
Closing due to inactivity, happy to re-open if you would like to take this on in the future

**Pull request closed without merging**